source/stylesheets/_blog.scss

```diff
@@ -111,6 +111,7 @@
 }
 
 .blog-posts__post-avatar {
+  background-color: $white;
   border-radius: 32px;
   box-shadow: 0 4.5px 13px 0 rgba(0,0,0,.1);
   display: block;
```

Comment:
This looks better if profile images are slow to load. Puts a bg color in the circle instead of it being transparent with the underlying border line showing through